dataset/companies/AI.yaml

```diff
@@ -0,0 +1,30 @@
+- Name: "OpenAI"
+  Description: "AI Research"
+  Website: "https://openai.com/"
+  Alternatives:
+    [
```

Comment:
Data should be in `yaml` format. Please run `npm run validate` before the commit.